bindings/python/src/pre_tokenizers.rs

```diff
@@ -220,6 +222,48 @@ impl PyWhitespaceSplit {
     }
 }
 
+/// Split PreTokenizer
+///
+/// This versatile pre-tokenizer splits using the provided pattern and
+/// according to the provided behavior. The pattern can be inverted by
+/// making use of the invert flag.
+///
+/// Args:
+///     pattern: Pattern:
+///         A pattern used to split the string. Usually a string or a Regex
+///
+///     behavior: SplitDelimiterBehavior:
+///         The behavior to use when splitting.
+///         Choices: "removed", "isolated", "merged_with_previous", "merged_with_next",
+///         "contiguous"
+///
+///     invert: bool:
+///         Whether to invert the pattern.
+#[pyclass(extends=PyPreTokenizer, module = "tokenizers.pre_tokenizers", name=Split)]
+#[text_signature = "(self, pattern, behavior, invert=False)"]
```

Comment:
not sure if this default signature is as you expected @n1t0 - I thought `pattern` and `behavior` is too general to give it default values, but `invert` could well default to `False`...

Reply:
Yes, it looks great! I was wondering if we could find a good default for `behavior` but I think you are right and it's better like this.